RecoTracker/ConversionSeedGenerators/python/ConversionStep_cff.py

```diff
@@ -310,13 +322,50 @@
 )
+_convLayerPairs = cms.EDProducer("SeedingLayersEDProducer",
+                                layerList = cms.vstring('BPix1+BPix2',
+                                                        'BPix2+BPix3',
+                                                        'BPix3+BPix4',
+
+                                                        'BPix1+FPix1_pos',
+                                                        'BPix1+FPix1_neg',
+                                                        'BPix2+FPix1_pos',
+                                                        'BPix2+FPix1_neg',
+                                                        'BPix3+FPix1_pos',
+                                                        'BPix3+FPix1_neg',
+
+                                                        'FPix1_pos+FPix2_pos',
+                                                        'FPix1_neg+FPix2_neg',
+                                                        'FPix2_pos+FPix3_pos',
+                                                        'FPix2_neg+FPix3_neg'
+                                                        ),
+
+                                BPix = cms.PSet(
+                                    hitErrorRZ = cms.double(0.006),
+                                    hitErrorRPhi = cms.double(0.0027),
+                                    TTRHBuilder = cms.string('TTRHBuilderWithoutAngle4PixelPairs'),
+                                    HitProducer = cms.string('siPixelRecHits'),
+                                    useErrorsFromParam = cms.bool(True),
+                                    skipClusters = cms.InputTag('convClusters'),
+                                    ),
+                                FPix = cms.PSet(
+                                    hitErrorRZ = cms.double(0.0036),
+                                    hitErrorRPhi = cms.double(0.0051),
+                                    TTRHBuilder = cms.string('TTRHBuilderWithoutAngle4PixelPairs'),
+                                    HitProducer = cms.string('siPixelRecHits'),
+                                    useErrorsFromParam = cms.bool(True),
+                                    skipClusters = cms.InputTag('convClusters'),
+                                    )
+)
+eras.trackingPhase2PU140.toReplaceWith(convLayerPairs, _convLayerPairs)
 
 photonConvTrajSeedFromSingleLeg.TrackRefitter = cms.InputTag('generalTracks')
 photonConvTrajSeedFromSingleLeg.primaryVerticesTag = cms.InputTag('firstStepPrimaryVertices')
 #photonConvTrajSeedFromQuadruplets.TrackRefitter = cms.InputTag('generalTracks')
 #photonConvTrajSeedFromQuadruplets.primaryVerticesTag = cms.InputTag('pixelVertices')
 eras.trackingLowPU.toModify(photonConvTrajSeedFromSingleLeg, primaryVerticesTag   = "pixelVertices")
 eras.trackingPhase1PU70.toModify(photonConvTrajSeedFromSingleLeg, primaryVerticesTag = "pixelVertices")
+eras.trackingPhase2PU140.toModify(photonConvTrajSeedFromSingleLeg, primaryVerticesTag = "pixelVertices")
```

Comment:
Technically this one will override the change in `PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py`. If you want `primaryVerticesTag` to stay as `offlinePrimaryVertices`, remove this line.

Anyway it is (especially nowadays) very bad habit to change parameters of a module defined elsewhere without cloning, i.e. we should (later) move all these to `PhotonConversionTrajectorySeedProducerFromSingleLeg_cfi.py`.